bucket/googlechrome.json

```diff
@@ -19,13 +19,21 @@
     "installer": {
         "script": "Expand-7zipArchive \"$dir\\chrome.7z\" -ExtractDir 'Chrome-bin' -Removal"
     },
-    "bin": "chrome.exe",
+    "bin": [
+        [
+            "chrome.exe",
+            "chrome",
+            "--user-data-dir=\"$dir\\User Data\""
+        ]
+    ],
     "shortcuts": [
         [
             "chrome.exe",
-            "Google Chrome"
+            "Google Chrome",
+            "--user-data-dir=\"$dir\\User Data\""
         ]
     ],
+    "persist": "User Data",
```

Comment:
No `User Data` migration for googlechrome?

Reply:
The non-portable manifest for Google Chrome stores data in the installation folder itself (as opposed to other non-portable manifests like Palemoon, which store in %AppData%), so no migration is needed here.

Reply:
I got a fresh installation after running the portable update... I'm sure my User Data for the non-portable Google Chrome was in `~\AppData\Local\Google\Chrome\User Data`

Reply:
The only difference between googlechrome.json and googlechrome-portable.json was the binary arguments and persist, which indicates the user data was being stored in the scoop\apps folder itself.

The %LocalAppData% folder is utilised by system-wide Google Chrome installation (which installs in C:\Program Files). I think this is due to the difference in the way the browser is installed, i.e. through running the installer, as opposed to Scoop which extracts the installer.

We should only migrate data from Scoop installations, and not system-wide installations (because they weren't installed through Scoop anyway).

Reply:
Not ture. I only have Chrome installed via scoop before.

For testing, I deleted all old chrome User Data, did a fresh installation by the current googlechrome.json, opened it, pinned a tab and modified some settings. I checked User Data was generated and saved in `~\AppData\Local\Google\Chrome\User Data`. Then I updated the manifest to this PR's and force updated it to the portable one. I got a fresh installation, no pinned tab, settings gone.

Reply:
Okay, I'll add a migration for this too then.

What about Firefox and Thunderbird?

Reply:
I'm not sure how Firefox and Thunderbird profile work, but if they have issue #4377 of persisting profile, then I would keep them with no migration.

Reply:
Added